examples/js/app.js

```diff
@@ -372,16 +372,23 @@
         calendarTypeIcon.className = iconClassName;
     }
 
+    function currentCalendarDate(format) {
+      var currentDate = moment([cal.getDate().getFullYear(), cal.getDate().getMonth(), cal.getDate().getDate()]);
+
+      return format ? currentDate.format(format) : currentDate;
```

Comment:
format is mandatory.  tenary operator is unnecessary.

Reply:
Yeah. I'll change.

`return currentDate.format(format);`